src/room/VideoPreview.tsx

```diff
@@ -62,25 +61,32 @@ export function VideoPreview({ matrixInfo, onUserChoicesChanged }: Props) {
   }, [settingsModalState]);
 
   // Fetch user media devices.
-  const mediaDevices = useMediaDevices();
-
+  const mediaDevices = useMediaAllDevicesWithLocalStorage();
+  const { permittedDevices, videoIn, audioIn } = mediaDevices;
```

Comment:
```suggestion
  const { permittedDevices, videoIn, audioIn } = useMediaAllDevicesWithLocalStorage();
```

Reply:
I wanted to shorten the return type that is why i am storing the full object in a seperate variable.

Reply:
Sorry, I am not sure I follow?